tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/ProcessBasicCheck.cs

```diff
@@ -41,12 +44,13 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
                 runtime = ProcessInfo.Runtime.NetFx;
             }
 
+            AnsiConsole.WriteLine(ModuleCheck("1"));
```

Comment:
One way you could have a dynamic check number is you could add a new method in Resources.cs like this:
```
public static string PrependAndIncrementCheck(string input) => $"{_checkNumber++}. {input}";
```

That way all of the strings you want to emit as checks just become `PrependAndIncrementCheck(string)`. Like this. WDYT?
```suggestion
            AnsiConsole.WriteLine(PrependAndIncrementCheck(ModuleCheck()));
```

Reply:
If you're going with that approach, and we know `ModuleCheck()` is always going to have a check number, I think you can run with this but go one step further:

```csharp
private static int _checkNumber = 0; // 👈 implied by Zach's suggestion

                                      // 👇 Automatically bump the number
public static string ModuleCheck() => $"{_checkNumber++}. Checking Modules Needed so the Tracer Loads:";

// Call this 👇 between "stages" to reset the check counts
public static void ResetChecks() => _checkNumber = 0;
```

This feels like best of all worlds to me :)